internal/components/componentlist.go

```diff
@@ -16,10 +16,11 @@ type ComponentList struct {
 }
 
 type Component struct {
-	Name          string
-	Namespace     string
-	URL           string
-	Configuration map[string]interface{}
+	Name              string
+	Namespace         string
+	URL               string
+	Configuration     map[string]interface{}
+	FlatConfiguration map[string]interface{}
```

Comment:
Please be aware coming refactoring in PR #404 :

The workspace factory is currently using a struct from the `internal`  pkg as part of its interface definition which isn't a good practise. We are changing this in #404 and will use the component from the `chart` package.